org.wso2.carbon.consent.mgt.core/pom.xml

```diff
@@ -81,5 +81,10 @@
             <groupId>org.wso2.carbon</groupId>
             <artifactId>org.wso2.carbon.logging</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Move to parent pom

Reply:
Moved to <dependencyManagement> in parent POM